python/test/gherkin_in_markdown_token_matcher_test.py

```diff
@@ -6,7 +6,16 @@
 from gherkin.gherkin_line import GherkinLine
 location = { 'line': 1, 'column': 1 }
 
-def test_it_matches_FeatureLine():
+def test_it_matches_FeatureLineH1():
```

Comment:
Outside the scope of this PR, but I recently found [pytest-describe](https://pypi.org/project/pytest-describe/) that would make it easier to match the describe style of the javascript variant

I really prefer it - it enables me to structure my specifications more logically.

One to think about...